keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -871,13 +902,13 @@ feature.
 
 ###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
 
-Use of the pemTrustAnchors projected volume type will impact the [existing
+Use of the ClusterTrustBundle projected volume type will impact the [existing
 SLOs](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/pod_startup_latency.md)
 that cover startup latency for stateless schedulable pods (similar to the
 existing effect from).
 
 The definition of "stateless" will also need to be updated to include pods that
-use pemTrustAnchors volumes, unless it is intentional that those SLOs exclude
+use ClusterTrustBundle volumes, unless it is intentional that those SLOs exclude
 projected token volumes.
 
 (Kubelet projected volumes seem problematic in general for the startup latency
```

Comment:
`### Troubleshooting` section needs to be filled out.

Reply:
`## Implementation History` section needs to be filled out.

Reply:
`What steps should be taken if SLOs are not being met to determine the problem?` needs an answer still.

Reply:
Done.